lib/pushmi_pullyu/logging.rb

```diff
@@ -0,0 +1,33 @@
+require 'logger'
+
+# PushmiPullyu::Logging is a standard Ruby logger wrapper
+module PushmiPullyu::Logging
```

Comment:
It occurred to me, if this is going to be a long-running daemon, we need to trap SIGHUP and swap out the logs to play nicely with log-rotation automation